microsoft-edge/toc.yml

```diff
@@ -782,126 +785,219 @@
+# if top-of-page title is longer, add it to displayName comma-delimited list of lookup keywords
     - name: WebView2
       items:
         - name: Introduction to Microsoft Edge WebView2
           href: webview2/index.md
-
-        - name: Get started
+          displayName:  # add alt lookup words, comma-delimited
+# -----------------------------------------------------------------------------
+        - name: "Tutorial: Create a WebView2 app"
           items:
             - name: Get started with WebView2
               href: webview2/get-started/get-started.md
+              displayName: 
 
-            - name: Get started with WebView2 in Win32 apps
+            - name: Win32
               href: webview2/get-started/win32.md
+              displayName: Get started with WebView2 in Win32 apps  # top-of-page title
 
-            - name: Get started with WebView2 in WinForms apps
+            - name: WinForms
               href: webview2/get-started/winforms.md
+              displayName: Get started with WebView2 in WinForms apps  # top-of-page title
 
-            - name: Get started with WebView2 in WinUI 2 (UWP) apps (public preview)
+            - name: WinUI 2 (UWP, public preview)
               href: webview2/get-started/winui2.md
+              displayName: Get started with WebView2 in WinUI 2 (UWP) apps (public preview)  # top-of-page title
 
-            - name: Get started with WebView2 in WinUI 3 (Windows App SDK)
+            - name: WinUI 3 (Windows App SDK)
               href: webview2/get-started/winui.md
+              displayName: Get started with WebView2 in WinUI 3 (Windows App SDK) apps  # top-of-page title
 
-            - name: Get started with WebView2 in WPF apps
+            - name: WPF
               href: webview2/get-started/wpf.md
+              displayName: Get started with WebView2 in WPF apps  # top-of-page title
+# -----------------------------------------------------------------------------
+        - name: Fundamentals
+          items:
+            - name: Sample Code
+              href: webview2/code-samples-links.md
+              displayName: Sample Code for WebView2  # top-of-page title
 
-        - name: WebView2 sample code
-          href: webview2/code-samples-links.md
+            - name: Differences between Microsoft Edge and WebView2
+              href: webview2/concepts/browser-features.md
+              displayName: 
 
-        - name: Concepts and How to
-          items:
-            - name: Automate and test WebView2 apps with Microsoft Edge Driver
-              href: webview2/how-to/webdriver.md
+            - name: Navigation events
+              href: webview2/concepts/navigation-events.md
+              displayName: Navigation events for WebView2 apps  # top-of-page title
 
-            - name: Get started debugging WebView2 apps
-              href: webview2/how-to/debug.md
+            - name: Web and native inter-communication
```

Comment:
Only nitpick is that while "Interop" isn't super beginner friendly, it is a fairly well-known word. I don't think I've ever seen inter-communication used in this way.

Reply:
changed, huge improvement.  A top priority is to use the standard language of the field; use word choice that the audience uses/understands.  In chats, I think we now have a firm grasp of this bucket and the 3 articles that will be in it: 
how 2-way calling works (existing doc); then how to call in one direction (existing doc), then how to call in the other direction (forthcoming doc).  It will be easier to improve the 4 involved TOC-node labels, and top-of-page titles, after the incoming article is added.